### DISCUSSION TITLE
is .net maui ready to use???

### DISCUSSION THREAD

**Discussion by zokirjanov** [dotnet/maui#16574] [General] [answered]
Hi guys!  I wonder,  is dotnet MAUI ready to use for real projects(like commercial app) or should i use Xamarin

**Comment by nor0x** [comment]
you will probably spend more time implementing workarounds or mess with tooling, than coding your actual app

**Comment by JRosanowski** [comment]
There are still a lot of workarounds required. On the other hand Xamarin Forms won't support iOS 17 or Android 14... 

https://dotnet.microsoft.com/en-us/platform/support/policy/xamarin
_Xamarin support will end on May 1, 2024 for all Xamarin SDKs. Android 13 and Xcode 14 SDKs (iOS and iPadOS 16, macOS 13) will be the final versions Xamarin will target._

**Comment by JohnHDev** [comment, marked as answer]
Regarding Xamarin Forms, you won't be able to sign new releases as of May 1 2024. Xamarin and Xamarin Forms are dead in the water.

Regarding .NET Maui, it depends what your app will do, what functionality you intend to use. I would experiment and see if you have any issues by creating quick POCs to test out the layouts you intend etc.

.NET Maui is far better than it was 6 months ago imo. We still aren't ready to migrate our Xamarin Forms apps yet however.

**Comment by Xyncgas** [comment]
A very popular api is getting user interface implementation by the community lately, the frameworks used are currently
qt : https://github.com/tonquer/picacg-qt
flutter : https://github.com/wgh136/PicaComic
I wonder if you can use MAUI to achieve it without resorting to MAUI.Blazor
